content/2017-09-19_pycon-cz-conference-wrap-up-summary.md

```diff
@@ -0,0 +1,199 @@
+Title: PyCon CZ Conference Wrap-Up Summary
+Date: 2017-09-19 20:19:00
```

Comment:
Note for myself: This should later change to something closer to the actual publishing date.

Reply:
@honzajavorek nezapomeň tady na to